stack.cabal

```diff
@@ -1,5 +1,5 @@
 name:                stack
-version:             0.1.2.1
+version:             0.1.2.9999
```

Comment:
Well, I wanted to mark this dev version and make sure we could rely on the locking functionality on our continuous integration infrastructure.  Is there a naming convention for dev versions a la GHC's "7.11.DATE".